### ISSUE TITLE
Online office editing not working after Nextcloud AIO 7.9.0 update

### ISSUE THREAD

**Issue by vawaver** [nextcloud/all-in-one#3959]
Greetings,
I'm reporting a bug with the online office editing feature in Nextcloud AIO 7.9.0. 
After updating to this version, I'm unable to open any Office documents online. 
The screen hangs when I try to open a document.

I'm running Nextcloud on an Ubuntu server 22.04.3. I haven't had this problem with any previous versions of Nextcloud AIO.

### Steps to reproduce
1. Click on the file that I want to open via Firefox
2. Loading document...

### Expected behavior 
The document will be opened for editing in the browser.

### Actual behavior 
The document is loading, screen stucks.

### Host OS 
Ubuntu 22.04.3

#### Nextcloud AIO version 
7.9.0
#### Current channel 
latest
#### Other valuable info 
![admin](https://github.com/nextcloud/all-in-one/assets/52236863/5703ee8f-8730-4384-9726-e76725ce8b8c)
![loading](https://github.com/nextcloud/all-in-one/assets/52236863/9a14ed7c-c3a1-4b87-b7c7-9a70fbf88097)
![logs](https://github.com/nextcloud/all-in-one/assets/52236863/1fdb7d5e-59e7-4f85-816e-9646e863df57)
[_nextcloud-aio-collabora_logs.txt](https://github.com/nextcloud/all-in-one/files/13749285/_nextcloud-aio-collabora_logs.txt)

**Comment by joshtrichards** [comment]
I'm not surprised you cannot edit since your Collabora contain is definitely offline.

Maybe something like this:
https://github.com/nextcloud/all-in-one/discussions/3261

**Comment by vawaver** [comment]
Translation of the Slovak instruction into English:

I found a solution to my problem. For easy operation, I used Portainer.

1. Stop all `nextcloud-aio` containers, except `nextcloud-aio-mastercontainer`  (in Portainer)
2. I removed all installation images except the image for `nextcloud-aio-mastercontainer` (in Portainer)
3. I started the installation again through the web browser.

Collabora is working again.
